> We'll probably need to wait a little bit before merging this, but there's only one final warning that I can see before this is ready to merge:
> 
> ```
> /tmp/llvm-project/build/tools/flang/docs/Source/FlangCommandLineReference.rst:176: WARNING: unknown document: 'DiagnosticsReference'
> ```
> 
> Once that gets fixed, we should be able to get this in.

This warning (and all others) are now fixed. But I see the following mlir warning that is probably caused because flang needs mlir tools.
```
/home/buildbot/as-worker-4/publish-sphinx-docs/llvm-project/mlir/tools/mlir-tblgen/OpDefinitionsGen.cpp:2956:38: warning: unused variable ‘operand’ [-Wunused-variable]
```

--- a/.github/workflows/docs.yml
+++ b/.github/workflows/docs.yml
@@ -162,8 +162,6 @@ jobs:
           TZ=UTC ninja -C polly-build docs-polly-html docs-polly-man
       - name: Build Flang docs
         if: steps.docs-changed-subprojects.outputs.flang_any_changed == 'true'
-        # TODO(boomanaiden154): Remove the SPHINX_WARNINGS_AS_ERRORS from the
-        # CMake invocation once the warnings in the flang docs build are fixed.
         run: |
-          cmake -B flang-build -GNinja -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_PROJECTS="clang;mlir;flang" -DLLVM_ENABLE_SPHINX=ON -DSPHINX_WARNINGS_AS_ERRORS=OFF ./llvm
+          cmake -B flang-build -GNinja -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_PROJECTS="clang;mlir;flang" -DLLVM_ENABLE_SPHINX=ON ./llvm
           TZ=UTC ninja -C flang-build docs-flang-html docs-flang-man

--- a/.github/workflows/docs.yml
+++ b/.github/workflows/docs.yml
@@ -166,4 +166,4 @@ jobs:
         if: steps.docs-changed-subprojects.outputs.flang_any_changed == 'true'
         run: |
           cmake -B flang-build -GNinja -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_PROJECTS="clang;mlir;flang" -DLLVM_ENABLE_SPHINX=ON ./llvm
-          TZ=UTC ninja -C flang-build docs-flang-html docs-flang-man
+          TZ=UTC ninja -C flang-build docs-flang-html

